doc/man5/config.pod

```diff
@@ -487,6 +488,10 @@ Ignored in set-user-ID and set-group-ID programs.
 The path to the directory with OpenSSL modules, such as providers.
 Ignored in set-user-ID and set-group-ID programs.
 
+=item B<OPENSSL_CONF_INCLUDE>
+
+The optional path to prepend to any file included using .include.
```

Comment:
s/file/path/ ?

Reply:
changed to The optional path to prepend to all .include paths.